libraries/ms-common/src/stm32f0xx/spi.c

```diff
@@ -11,14 +11,10 @@ typedef struct {
 
 static SPIPortData s_port[SPI_MCU_NUM_PORTS] = {
   {
-    .rcc_cmd = RCC_APB2PeriphClockCmd,
-    .periph = RCC_APB2Periph_SPI1,
-    .base = SPI1
+      .rcc_cmd = RCC_APB2PeriphClockCmd, .periph = RCC_APB2Periph_SPI1, .base = SPI1,
```

Comment:
So the comma only makes the following change
```c
{ .hello = 1, .test = 1 };
// vs
{
  .hello = 1, .test = 1,
};
```
The multi-line initializer
{
  .hello = 1,
  .test = 1,
};
```
will only happen if the arguments/parameters to the initializer exceed our column limit of 100 characters.